pkg/providers/instance/instance_test.go

```diff
@@ -149,6 +150,48 @@ func TestGetZone(t *testing.T) {
 	}
 }
 
+func TestSetNodePoolNameTag(t *testing.T) {
+	tc := []struct {
+		testName      string
+		nodeClaim     *corev1beta1.NodeClaim
+		tags          map[string]*string
+		expectedError string
+		expectedTags  map[string]*string
+	}{
+		{
+			testName: "happy case",
+			nodeClaim: &corev1beta1.NodeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						corev1beta1.NodePoolLabelKey: "pool1",
+					},
+				},
+			},
+			tags: map[string]*string{},
+			expectedTags: map[string]*string{
+				NodePoolTagKey: to.Ptr("pool1"),
+			},
+		},
+		{
+			testName: "Node Claim is missing nodepool label",
+			nodeClaim: &corev1beta1.NodeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"key1": "val1",
+					},
+				},
+			},
+			tags:         map[string]*string{},
+			expectedTags: map[string]*string{},
+		},
+	}
+
+	for _, c := range tc {
+		setNodePoolNameTag(c.tags, c.nodeClaim)
```

Comment:
We should get coverage fine from the acceptance tests you added imo we don't need this :)

Reply:
The unit test for the function is now removed.